History.md

```diff
@@ -1,3 +1,6 @@
+0.9.0 / TBA
+ * perf: Improve overall performance
```

Comment:
Would help if we can quantify the performance improvement here.
You've done a great job and I'd like people to know that.

"Improve overall performance, up to 10x in some functionalities." might sell better.

Reply:
Will have a thought on this.